lapce-app/src/palette/kind.rs

```diff
@@ -24,13 +25,7 @@ impl PaletteKind {
             // PaletteKind::GlobalSearch => "?",
             PaletteKind::Workspace => ">",
             PaletteKind::Command => ":",
-            PaletteKind::File
```

Comment:
I would leave this so that you don't forget if it has a kind symbol when a new palette kind is added.